packages/flutter/lib/src/services/system_chrome.dart

```diff
@@ -77,17 +78,141 @@ enum SystemUiOverlay {
+  final Color statusBarColor;
+
+  /// The brightness of top status bar.
+  /// 
+  /// Only honored in iOS .
+  final Brightness statusBarBrightness;
+
+  /// The brightness of the top status bar icons.
+  /// 
+  /// Only honored in Android version M and greater.
+  final Brightness statusBarIconBrightness;
+
+  /// The brightness of the system navigation bar icons.
+  /// 
+  /// Only honored in Android versions O and greater.
+  final Brightness systemNavigationBarIconBrightness;
+
+  /// Convert this event to a map for serialization.
+  Map<String, dynamic> toMap() {
+    return <String, dynamic>{
+      'systemNavigationBarColor': systemNavigationBarColor?.value,
+      'systemNavigationBarDividerColor': systemNavigationBarDividerColor?.value,
+      'statusBarColor': statusBarColor?.value,
+      'statusBarBrightness': statusBarBrightness?.toString(),
+      'statusBarIconBrightness': statusBarIconBrightness?.toString(),
+      'systemNavigationBarIconBrightness': systemNavigationBarIconBrightness?.toString(),
+    };
+  }
+
+  /// Creates a copy of this theme with the given fields replaced with new values.
+  SystemUiOverlayStyle copyWith({
+    Color systemNavigationBarColor,
+    Color systemNavigationBarDividerColor,
+    Color statusBarColor,
+    Brightness statusBarBrightness,
+    Brightness statusBarIconBrightness,
+    Brightness systemNavigationBarIconBrightness,
+  }) {
+    return new SystemUiOverlayStyle(
+      systemNavigationBarColor: systemNavigationBarColor ?? this.systemNavigationBarColor,
+      systemNavigationBarDividerColor: systemNavigationBarDividerColor ?? this.systemNavigationBarDividerColor,
+      statusBarColor: statusBarColor ?? this.statusBarColor,
+      statusBarIconBrightness: statusBarIconBrightness ?? this.statusBarIconBrightness,
+      statusBarBrightness: statusBarBrightness ?? this.statusBarBrightness,
+      systemNavigationBarIconBrightness: systemNavigationBarIconBrightness ?? this.systemNavigationBarIconBrightness,
+    );
+  }
+
+  @override
+  int get hashCode => 
+    hashValues(
+      systemNavigationBarColor,
+      systemNavigationBarDividerColor,
+      statusBarColor,
+      statusBarBrightness,
+      statusBarIconBrightness,
+      systemNavigationBarIconBrightness,
+    );
+
+  @override
+  bool operator ==(dynamic other) {
+    if (other.runtimeType != runtimeType)
+      return false;
+    final SystemUiOverlayStyle typedOther = other;
+    return typedOther.systemNavigationBarColor == systemNavigationBarColor
+      && typedOther.systemNavigationBarDividerColor == systemNavigationBarDividerColor
+      && typedOther.statusBarColor == statusBarColor
+      && typedOther.statusBarIconBrightness == statusBarIconBrightness
+      && typedOther.statusBarBrightness == statusBarBrightness
+      && typedOther.systemNavigationBarIconBrightness == systemNavigationBarIconBrightness;
+  }
 }
 
 List<String> _stringify(List<dynamic> list) {
```

Comment:
This function could be written or just inlined as
```
list.map((item) => item.toString()).toList()
```
The `toList()` might not be needed either, depending the implementation of invokeMethod.

Reply:
I didn't write this code so I'm going to leave it as is